### PR TITLE
Sort and optimize python imports

### PR DESCRIPTION
**Pull request by uelei**

### CODE REVIEW

File: modules/lang/python/config.el [hunk 1]
```diff
@@ -118,14 +124,14 @@ called.")
 
   (map! :localleader
         :map nose-mode-map
-        :prefix "t"
+        (:prefix ("t"."tests")
```
[the remark this answers is not in the document]
Comment (hlissner) [reply modules/lang/python/config.el]: Also, please remove the parentheses "grouping" around :prefix. It's unnecessary in this case, and the block isn't correct indented in either case.

File: modules/lang/python/config.el [hunk 2]
```diff
@@ -134,13 +140,13 @@ called.")
   (map! :after python
         :localleader
         :map python-mode-map
-        :prefix "t"
+        (:prefix ("t"."tests")
```
Comment (hlissner) [inline modules/lang/python/config.el]: Please remove the parentheses "grouping" around :prefix. It's unnecessary in this case.

**Comment by uelei** [comment]
some corrections made ! if need to change anything else let me know ! (sorry my mistakes i dont know progm in elisp ) !

**Comment by hlissner** [comment]
@uelei Thanks for the help! Sorry to trouble you with this, but could you rebase this PR onto the `straight` branch and squash the commits into one (to slim down the PRs commit history)? `straight` will be merged into develop soon, and it'd be a big help if PRs between now and then targeted it instead.

**Comment by uelei** [comment]
I mess up the commits, i just made a new fresh PR ! https://github.com/hlissner/doom-emacs/pull/1658

**Pull request closed without merging**